Gemfile

```diff
@@ -8,7 +10,13 @@ end
 
 platforms :jruby do
   gem 'jruby-openssl'
-  gem 'activerecord-jdbcsqlite3-adapter'
+  if rails_version == 'edge' || rails_version.match(/5\.\d+\.\d+/)
+    gem 'activerecord-jdbcsqlite3-adapter',
+        git: 'https://github.com/jruby/activerecord-jdbc-adapter.git',
+        branch: 'rails-5'
```

Comment:
Yup. See first comment. Still in process of testing.  See comment I am about to type